### PR TITLE
e2e-Normalize Test Scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
@rmstar @itsankit-google Please review.
cc @sawantpritam

### CODE REVIEW

File: e2e-test/src/test/resources/pluginParameters.properties [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+overrideDataType=float
+readRecursivePath=testdata/file/RECURSIVE_TEST
+readRecursivePathSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"tablename","value":"string"},{"key":"offset","value":"string"}]
+fileMaxSplitSize=120
+regexPathFilter=.+TEST.csv
+incorrectRegexPathFilter=/tZY
+filePluginOutputFolder=file-plugin-output
+skipHeaderTrue=true
+fileSinkPathSuffix=yyyy-MM-dd-HH-mm-ss
```
Comment (rmstar) [inline e2e-test/src/test/resources/pluginParameters.properties]: Remove configs that are ununused/not relevant to normalize plugin tests

Comment (itsmekumari) [reply e2e-test/src/test/resources/pluginParameters.properties]: Removed the unused configs.

File: e2e-test/src/test/features/BigQueryToNormalize.feature [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count is equal to records transferred to target BigQuery table
```
Comment (rmstar) [inline e2e-test/src/test/features/BigQueryToNormalize.feature]: verify that data written to the sink is correct

File: e2e-test/src/test/features/BigQueryToNormalize.feature [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+  @BQ_SOURCE_NORMALIZE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred from BigQuery to GCS sink plugin successfully with Normalized fields
```
Comment (rmstar) [inline e2e-test/src/test/features/BigQueryToNormalize.feature]: no need to test different combinations of sources and sinks. these tests should focus on transpose plugin functionality.

File: e2e-test/src/test/features/BigQueryToNormalize.feature (hunk 2 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/BigQueryToNormalize.feature]: As per the changes suggested in [Distinct PR](https://github.com/cdapio/hydrator-plugins/pull/1644) included plugins from hydrator-plugin repo only. Hence removed the scenario.

File: e2e-test/src/test/features/BigQueryToNormalize.feature (hunk 3 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/BigQueryToNormalize.feature]: Done.

**Review by rmstar — APPROVED** [review]

**Pull request merged**